src/adu-shell/src/main.cpp

```diff
@@ -145,8 +145,9 @@ int ParseLaunchArguments(const int argc, char** argv, ADUShell_LaunchArguments*
             launchArgs->logFile = optarg;
             break;
 
+        // clang-format off
```

Comment:
why clang-format off?

Reply:
i'm fine with that style, and would look better than two clang-format comment lines.  alternately, remove the need for the braces (move stack var defn outside of switch scope)

Reply:
removed clang-format off and reformatted.